flink-kubernetes/src/main/resources/META-INF/NOTICE

```diff
@@ -10,14 +10,14 @@ This project bundles the following dependencies under the Apache Software Licens
 - com.fasterxml.jackson.core:jackson-core:2.10.1
 - com.fasterxml.jackson.core:jackson-databind:2.10.1
 - com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.10.1
-- com.github.mifmif:generex:1.0.2
```

Comment:
I think we still bundle `com.github.mifmif:generex:1.0.2`. Hence we cannot remove this entry.

Reply:
Currently, we do not bundle `com.github.mifmif:generex:1.0.2` into the flink-dist jar. Because we only have `<include>com.mifmif:*</include>` in flink-kubernetes/pom.xml.
Maybe it is typo and should be `com.github.mifmif`. I will fix it.

Reply:
You are right. If it worked before, why do we need to bundle `com.github.mifmif`? If we don't need it, then we can also remove the shading of this dependency.

Reply:
Because we do not go into the code path which are using `com.mifmif.common.regex.Generex` in fabric8 kubernetes-client. So do you suggest to completely remove the `com.github.mifmif` dependency?

Reply:
If we don't use this code, then we don't need to bundle it in the resulting artifact.

Reply:
I prefer bundling `com.github.mifmif` into the flink-dist jar. It is very small and we may go into the code path in the future, especially when introducing some flexible features(e.g. pod template).

Reply:
Yeah, i will remove the dependency.